numpy/_core/arrayprint.py

```diff
@@ -35,33 +35,17 @@
 from .umath import absolute, isinf, isfinite, isnat
 from . import multiarray
 from .multiarray import (array, dragon4_positional, dragon4_scientific,
-                         datetime_as_string, datetime_data, ndarray,
-                         set_legacy_print_mode)
+                         datetime_as_string, datetime_data, ndarray)
 from .fromnumeric import any
 from .numeric import concatenate, asarray, errstate
 from .numerictypes import (longlong, intc, int_, float64, complex128,
                            flexible)
 from .overrides import array_function_dispatch, set_module
+from ._printoptions import _format_options
```

Comment:
If my suggestion above of not "doubly privating" code is adapted, this would be `from .printoptions import format_options`